<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1173</width>
    <height>761</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="statePlainTextEdit">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>510</y>
      <width>621</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget1">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>10</y>
      <width>86</width>
      <height>284</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>设备地址</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="SlaveAddressLineEdit">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="openButton">
       <property name="text">
        <string>连接设备</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>轮询速度</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="pollComboBox"/>
     </item>
     <item>
      <widget class="QPushButton" name="pollPushButton">
       <property name="text">
        <string>轮询</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopPushButton">
       <property name="text">
        <string>停止</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLCDNumber" name="timelcdNumber">
    <property name="geometry">
     <rect>
      <x>33</x>
      <y>657</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="digitCount">
     <number>8</number>
    </property>
   </widget>
   <widget class="QTableWidget" name="deviceInfoTableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>541</width>
      <height>625</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="gridStyle">
     <enum>Qt::SolidLine</enum>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <property name="rowCount">
     <number>17</number>
    </property>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>19</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>177</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>21</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>35</number>
    </attribute>
    <attribute name="verticalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <row>
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignTrailing|AlignVCenter</set>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <column>
     <property name="text">
      <string>项目</string>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignLeading|AlignVCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>采集值</string>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>15</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>单位</string>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>15</pointsize>
      </font>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>设备型号:0x9A</string>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>生成日期:0x9D</string>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>设备ID:0x99</string>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string>生产厂地:0x9C</string>
     </property>
    </item>
    <item row="3" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="4" column="0">
     <property name="text">
      <string>芯片ID:0xAD</string>
     </property>
    </item>
    <item row="4" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="5" column="0">
     <property name="text">
      <string>最小输入电压:0xA0</string>
     </property>
    </item>
    <item row="5" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="5" column="2">
     <property name="text">
      <string>V</string>
     </property>
    </item>
    <item row="6" column="0">
     <property name="text">
      <string>最大输入电压:0xA1</string>
     </property>
    </item>
    <item row="6" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="6" column="2">
     <property name="text">
      <string>V</string>
     </property>
    </item>
    <item row="7" column="0">
     <property name="text">
      <string>最小输入电流:0xA2</string>
     </property>
    </item>
    <item row="7" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="7" column="2">
     <property name="text">
      <string>A</string>
     </property>
    </item>
    <item row="8" column="0">
     <property name="text">
      <string>最大输入功率:0xA3</string>
     </property>
    </item>
    <item row="8" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="8" column="2">
     <property name="text">
      <string>W</string>
     </property>
    </item>
    <item row="9" column="0">
     <property name="text">
      <string>最小输出电压:0xA4</string>
     </property>
    </item>
    <item row="9" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="9" column="2">
     <property name="text">
      <string>V</string>
     </property>
    </item>
    <item row="10" column="0">
     <property name="text">
      <string>最大输出电压:0xA5</string>
     </property>
    </item>
    <item row="10" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="10" column="2">
     <property name="text">
      <string>V</string>
     </property>
    </item>
    <item row="11" column="0">
     <property name="text">
      <string>最大输出电流:0xA6</string>
     </property>
    </item>
    <item row="11" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="11" column="2">
     <property name="text">
      <string>A</string>
     </property>
    </item>
    <item row="12" column="0">
     <property name="text">
      <string>最大输出功率:0xA7</string>
     </property>
    </item>
    <item row="12" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="12" column="2">
     <property name="text">
      <string>W</string>
     </property>
    </item>
    <item row="13" column="0">
     <property name="text">
      <string>最大环境温度:0xA8</string>
     </property>
    </item>
    <item row="13" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="13" column="2">
     <property name="text">
      <string>℃</string>
     </property>
    </item>
    <item row="14" column="0">
     <property name="text">
      <string>最小环境温度:0xA9</string>
     </property>
    </item>
    <item row="14" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="14" column="2">
     <property name="text">
      <string>℃</string>
     </property>
    </item>
    <item row="15" column="0">
     <property name="text">
      <string>90Vac效率::0xAA</string>
     </property>
    </item>
    <item row="15" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="15" column="2">
     <property name="text">
      <string>%</string>
     </property>
    </item>
    <item row="16" column="0">
     <property name="text">
      <string>264Vac效率:0xAB</string>
     </property>
    </item>
    <item row="16" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="16" column="2">
     <property name="text">
      <string>%</string>
     </property>
    </item>
   </widget>
   <widget class="QTableWidget" name="workDataTableWidget">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>491</width>
      <height>301</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>25</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>160</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>27</number>
    </attribute>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <column>
     <property name="text">
      <string>项目</string>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignLeading|AlignVCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>采集值</string>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>15</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>单位</string>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>15</pointsize>
      </font>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>输入功率</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>W</string>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>输入电流:0x89</string>
     </property>
    </item>
    <item row="1" column="2">
     <property name="text">
      <string>A</string>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>输入电压:0x88</string>
     </property>
    </item>
    <item row="2" column="2">
     <property name="text">
      <string>V</string>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string>输出功率:0x96</string>
     </property>
    </item>
    <item row="3" column="2">
     <property name="text">
      <string>W</string>
     </property>
    </item>
    <item row="4" column="0">
     <property name="text">
      <string>输出电流:0x8C</string>
     </property>
    </item>
    <item row="4" column="2">
     <property name="text">
      <string>A</string>
     </property>
    </item>
    <item row="5" column="0">
     <property name="text">
      <string>输出电压:0x8B</string>
     </property>
    </item>
    <item row="5" column="2">
     <property name="text">
      <string>V</string>
     </property>
    </item>
    <item row="6" column="0">
     <property name="text">
      <string>环境温度:0x8D</string>
     </property>
    </item>
    <item row="6" column="2">
     <property name="text">
      <string>℃</string>
     </property>
    </item>
    <item row="7" column="0">
     <property name="text">
      <string>次级温度:0x8E</string>
     </property>
    </item>
    <item row="7" column="2">
     <property name="text">
      <string>℃</string>
     </property>
    </item>
    <item row="8" column="0">
     <property name="text">
      <string>初级温度:0x8F</string>
     </property>
    </item>
    <item row="8" column="2">
     <property name="text">
      <string>℃</string>
     </property>
    </item>
    <item row="9" column="0">
     <property name="text">
      <string>风扇转速:0x90</string>
     </property>
    </item>
    <item row="9" column="2">
     <property name="text">
      <string>R/Min</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="updataPushButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>650</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>更新数据</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>650</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>清除数据</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>650</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>清除数据</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cleanDataPushButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>650</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>清除数据</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_6">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>360</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Write Byte</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_7">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>400</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Write Word</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>360</y>
      <width>75</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_7">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>400</y>
      <width>75</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_8">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>400</y>
      <width>75</width>
      <height>21</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>LH</string>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::LogicalMoveStyle</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_9">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>360</y>
      <width>75</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_10">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>400</y>
      <width>75</width>
      <height>21</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>Hh</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>310</y>
      <width>81</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QLineEdit" name="SendByteLineEdit"/>
     </item>
     <item>
      <widget class="QLineEdit" name="readByteLineEdit"/>
     </item>
     <item>
      <widget class="QLineEdit" name="readWordLineEdit"/>
     </item>
     <item>
      <widget class="QLineEdit" name="BlockReadlineEdit"/>
     </item>
     <item>
      <widget class="QLineEdit" name="BlockWriteLineEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="BlockWritedatalineEdit">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>478</y>
      <width>401</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="BlockReadCountlineEdit">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>440</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="SMbusPushButton">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>330</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>310</y>
      <width>111</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1,1,1,1">
     <property name="leftMargin">
      <number>15</number>
     </property>
     <item>
      <widget class="QRadioButton" name="radioButton">
       <property name="text">
        <string>Send Byte</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="text">
        <string>Read Byte</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_3">
       <property name="text">
        <string>Read Word</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_4">
       <property name="text">
        <string>Block Read</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_5">
       <property name="text">
        <string>Block Write</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="checkByteCountPushButton">
    <property name="geometry">
     <rect>
      <x>1054</x>
      <y>434</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>检测</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>440</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>当不清楚字长时可点击检测</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1173</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuIIC">
    <property name="title">
     <string>IIC上位机</string>
    </property>
   </widget>
   <addaction name="menuIIC"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
